SPEC.md

```diff
@@ -118,6 +118,13 @@ Image Format TODO
 * Define security requirements for a container. In particular is any isolation of users required between containers? What user does each application run under and can this be root (i.e. "real" root in the host).
 * Define how apps are supposed to communicate; can they/do they 'see' each other (a section in the apps perspective would help)?
 
+### Pod Manifest
```

Comment:
>> Pods
>> In Kubernetes, rather than individual containers, pods are the smallest deployable units that can be created, scheduled, and managed.

> The only thing fuzzy here is "container", but that's well-known to be a weasel world and I'll touch on that more below. Perhaps "self-contained application environment" or so would be better - I would actually expect the language to change here in future to clarify this (sadly they don't define "container" in their glossary!). Otherwise, this captures exactly what we're trying to propose.

I'm not so sure. This language feels cherry-picked because the entire Pods section of the Kubernetes documentation is predicated upon the model being used (manipulating multiple containers simultaneously).
![Kubernetes Architecture](https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/docs/architecture.png)

>> A pod models an application-specific "logical host" in a containerized environment. It may contain one or more containers which are relatively tightly coupled -- in a pre-container world, they would have executed on the same physical or virtual host.

> Again, "containers" is weasly, but the rest is completely accurate.

"It may contain one or more containers" ... an example of how this model is just not the same. As App Container is _already_ unique, I would rather approach this from the other direction. Decompose and define the pieces (create/extend the model) first, then incorporate shared language where appropriate. Either appc ends up with its own well-defined terms or it might end up matching an existing model (in which case we should use shared language). Hunting for exiting language to use on a different model seems like a recipe for confusion. It might also lead to needing additional language in documentation to contrast the App Container model from the model the language is being borrowed from (as "pods" already exists in the space, I feel this is likely).